### PR TITLE
Dala 389-2 deploy to preview for real

### PR DESCRIPTION
**Pull request by MarcTM01**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [ ] ~At least one E2E Test exists testing the new feature~
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [x] The new version is deployed to the dev server using this branch
  - [x] It's verified that this version actually is the one deployed (check actuator/info for branch name and commit id!)
  - [x] The new feature is manually used/tested/observed on dev server
  - [x] All implemented Social Logins have been tested manually in the UI
- [ ] ~There is at least one picture for each story, which was created before coding has started~
- [x] If any work on the UI is to be merged, those changes were also documented in the Figma 
- [x] The local Dev stack still works: execute `startDevelopmentStack.sh`, npm serve, npm teste2e and execute Cypress Tests locally

### CODE REVIEW

File: dataland-inbound-proxy/templates/letsencrypt.template [hunk 1]
```diff
@@ -13,6 +13,6 @@ server {
     }
 
     location / {
-        return 301 https://$host$request_uri;
+        return 301 https://${PROXY_PRIMARY_URL}$request_uri;
```
Comment (ppuetsch) [inline dataland-inbound-proxy/templates/letsencrypt.template:16]: Damit machen wir das redirect auf die richtige URL unverschlüsselt. 

Ich bin mir unsicher - aber wäre es vielleicht besser ` return 301 https://$host$request_uri;` zu lassen - dann wird unter gleicher URL auf https geupgraded - und dann wird von da aus das redirect auf die richtige URL ausgegeben?

File: dataland-inbound-proxy/templates/dataland-development.template [hunk 2]
```diff
@@ -48,4 +48,17 @@ server {
     include utils/general.conf;
     include utils/error.conf;
 
+}
+
+server {
+   listen                  443 ssl http2 default_server;
+   listen                  [::]:443 ssl http2 default_server;
+   root                    /var/www/;
```
Comment (ppuetsch) [inline dataland-inbound-proxy/templates/dataland-development.template:56]: brauchen wir `   root                    /var/www/;` hier?

File: dataland-inbound-proxy/templates/dataland-production.template [hunk 3]
```diff
@@ -45,4 +45,17 @@ server {
     # additional config
     include utils/general.conf;
     include utils/error.conf;
+}
+
+server {
+   listen                  443 ssl http2 default_server;
+   listen                  [::]:443 ssl http2 default_server;
+   root                    /var/www/;
```
Comment (ppuetsch) [inline dataland-inbound-proxy/templates/dataland-production.template:53]: brauchen wir `   root                    /var/www/;`hier?

File: dataland-inbound-proxy/templates/letsencrypt.template (hunk 1 above)
Comment (MarcTM01) [reply dataland-inbound-proxy/templates/letsencrypt.template:16]: Ich sehe den Mehrwert daran nicht. Dadurch würden wir ja nur die Information: gehe bitte nicht nach www.dataland.com sondern nach dataland.com verschlüsseln. Das ist ja jetzt nicht sicherheitskritisch. Außerdem würde dadurch eine extra request entstehen, die wir uns auch sparen können.

File: dataland-inbound-proxy/templates/dataland-development.template (hunk 2 above)
Comment (MarcTM01) [reply dataland-inbound-proxy/templates/dataland-development.template:56]: Prüfe ich

Comment (MarcTM01) [reply dataland-inbound-proxy/templates/dataland-development.template]: Removed

File: dataland-inbound-proxy/templates/dataland-production.template (hunk 3 above)
Comment (MarcTM01) [reply dataland-inbound-proxy/templates/dataland-production.template]: Removed

File: dataland-inbound-proxy/templates/letsencrypt.template (hunk 1 above)
Comment (ppuetsch) [reply dataland-inbound-proxy/templates/letsencrypt.template:16]: ok

File: dataland-inbound-proxy/templates/dataland-development.template (hunk 2 above)
Comment (ppuetsch) [reply dataland-inbound-proxy/templates/dataland-development.template]: ok

File: dataland-inbound-proxy/templates/dataland-production.template (hunk 3 above)
Comment (ppuetsch) [reply dataland-inbound-proxy/templates/dataland-production.template]: ok

**Pull request merged**